xl_auth/templates/users/view.html

```diff
@@ -0,0 +1,85 @@
+<!-- users/view.html -->
+{% extends "layout.html" %}
+{% block content %}
+    <h1>{{ _('View User \'%(email)s\'', email=user.email) }}</h1>
+    <br/>
+    <div class="panel panel-default">
+        <dl class="dl-horizontal">
+            <dt>{{ _('Full name') }}:</dt>
+            <dd>{{ user.full_name }}</dd>
+
+            <dt>{{ _('Last Login At') }}:</dt>
+            <dd>{{ '-' if not user.last_login_at else user.last_login_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}</dd>
+
+            <dt>{{ _('Active Account') }}:</dt>
+            <dd>{{ _('Yes') if user.is_active else _('No') }}</dd>
+
+            <dt>{{ _('Administrator') }}:</dt>
+            <dd>{{ _('Yes') if user.is_admin else _('No') }}</dd>
+
+            <dt>{{ _('Cataloguing Administrator') }}:</dt>
+            <dd>{{ _('Yes') if user.is_cataloging_admin else _('No') }}</dd>
+
+            <dt>{{ _('Last Modified') }}:</dt>
+            <dd>
+                {{ user.modified_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=user.modified_by.id) }}">
+                    {{ user.modified_by.full_name }}
+                </a>
+            </dd>
+
+            <dt>{{ _('Created At') }}:</dt>
+            <dd>
+                {{ user.created_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=user.created_by.id) }}">
+                    {{ user.created_by.full_name }}
+                </a>
+            </dd>
+        </dl>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>{{ _('Permissions') }}</strong>
+        </div>
+        <table class="table table-hover">
+            <thead>
+            <tr>
+                <th class="col-md-1">{{ _('Collection') }}</th>
+                <th class="col-md-3 word-break-all">{{ _('Registrant') }}</th>
+                <th class="col-md-2 word-break-all">{{ _('Cataloger') }}</th>
+                <th class="col-md-3 word-break-all">{{ _('Cataloguing Administrator') }}</th>
+                <th class="col-md-1">{{ _('Active Collection') }}</th>
+                <th class="col-md-2">{{ _('Last Modified') }}</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for permission in user.permissions | sort(attribute='collection.code') %}
```

Comment:
@mxtthias When viewing a user's permissions, the following filtering rules should apply:
* [ ] unfiltered viewed as the user itself
* [ ] unfiltered when viewed as a superadmin
* [ ] filtered to only include a) permissions where the user has ``cataloging_admin`` set to true or b) any permissions for collections where the viewer has rights as a cataloging admin

Reply:
:+1: